README.md

```diff
@@ -80,7 +77,7 @@ singularity exec "$URI" snakefmt --help
 The above will use the latest version. If you want to specify a version then use a [tag][dockerhub] like so.
 
 ```sh
-VERSION="0.1.3"
+VERSION="x.x.x"
```

Comment:
keeping this with a valid version number is much clearer

Reply:
but at least it forces the user to specify which version they want to use. its obvious to me you replace that by your version of interest. we could add a `poetry version | grep -oE "([0-9]\.)+[0-]"` for latest version instead

Reply:
nm